tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -0,0 +1,115 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This kubevirt test relies on the upstream test code, which is downstreamed as virt-tests.
+#          This is the part running on agent node.
+# Maintainer: Nan Zhang <nan.zhang@suse.com>
+
+use base multi_machine_job_base;
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+    my $sut_ip = get_required_var('SUT_IP');
+
+    set_var('AGENT_IP', $sut_ip);
+    bmwqemu::save_vars();
+
+    # Synchronize the server & agent nodes
+    barrier_create('KUBEVIRT_TEST_SETUP', 2);
```

Comment:
Duplicate with kubevirt_barriers.pm ?

Reply:
Will remove one of barrier_create functions in both modules after verification run confirmed.